pysb/examples/michment.py

```diff
@@ -0,0 +1,26 @@
+from pysb import *
```

Comment:
Sure, that's fine. Maybe just call this one michment_massaction.py or something.